core/indexer/workItems/interface.go

```diff
@@ -0,0 +1,39 @@
+package workItems
```

Comment:
IsInterfaceNil() method is not added in these interfaces. We don't need this method later ?

Reply:
done.